src/access_log.jl

```diff
@@ -0,0 +1,110 @@
+"""
+    logfmt"..."
+
+Parse an [NGINX-style log format string](https://nginx.org/en/docs/http/ngx_http_log_module.html#log_format)
+and return a function mapping `(io::IO, http::HTTP.Stream) -> body` suitable for passing to
+[`HTTP.listen`](@ref) using the `access_logfmt` keyword argument.
+
+The following variables are currently supported:
+
+ - `\$http_name`: arbitrary request header (with `-` replaced with`_`, e.g. `http_user_agent`)
+ - `\$sent_http_name`: arbitrary response header (with `-` replaced with`_`)
+ - `\$request`: the request line, e.g. `GET /index.html HTTP/1.1`
+ - `\$request_method`: the request method
+ - `\$request_uri`: the request URI
+ - `\$remote_addr`: client adress
+ - `\$remote_port`: client port
+ - `\$remote_user`: user name supplied with the Basic authentication
+ - `\$server_protocol`: server protocol
+ - `\$time_iso8601`: local time in ISO8601 format
+ - `\$time_local`: local time in Common Log Format
+ - `\$status`: response status code
+ - `\$body_bytes_sent`: number of bytes in response body
+
+## Examples
+```julia
+logfmt"[\$time_iso8601] \\"\$request\\" \$status" # [2021-05-01T12:34:40+0100] "GET /index.html HTTP/1.1" 200
+
+logfmt"\$remote_addr \\"\$http_user_agent\\"" # 127.0.0.1 "curl/7.47.0"
+```
+"""
```

Comment:
Is there a way wr can write `raw"""` for docstrings?
The multiple layers of escaping does my head-in.
and I know a surprising number of people read docstrings not through REPL or web, but by looking at the source

Reply:
You still need `\\` for `\` etc, but not for `$` so updated.